### DISCUSSION TITLE
[POLL] PLC4X - Hackathon - 16 / 17th December 2023 - Frankfurt/Main - Germany

### DISCUSSION THREAD

**Discussion by ottlukas** [apache/plc4x#1082] [Polls]
The PLC4X Hackathon has the following goals:

1. Preparing the next release - "The journey of a thousand miles begins with a single step." - Lao Tzu
2. The hackathon is a great place to meet new people and learn new skills. Come and join the community.
3. Failure is a part of the learning process. Come to the hackathon and take some risks.
4. Have fun and some drinks - The hackathon is a challenge, but it's also a lot of fun. Come and see what you can accomplish.

The hackathon is a chance to build something that you're passionate about. Come and make a difference.
This is your chance to make your mark on the world. Come to the hackathon and build something amazing.

**Comment by splatch** [comment]
I am positive to participate in hackathon, yet I am a bit afraid about date. Its a week before Christmas, couple of days before EU goes into its winter sleep for about 2-4 weeks. ;) Any significant work, which could be impactful to upcoming release, will be hard to complete over this period of time. Also its pretty late for getting release itself. Assuming that there will be some changes/improvements to API, it will be too late to get it done before end of year.

**Reply by ottlukas** [reply]
Preparing the next release does not mean that we will release during the Hackathon, but have a clearer pathway towards the release and will see what happens ;-)

**Comment by chrisdutz** [comment]
Actually I also think we should possibly start the release train now so we have something to report for Community Over Code

**Comment by chrisdutz** [comment]
But we could concentrate on discussing the "road to 1.0.0" ;-)

**Comment by hutcheb** [comment]
It sounds like fun, count me in.

**Comment by hongjinlin** [comment]
"The journey of a thousand miles begins with a single step.", I know the saying, haha.
I would like to participate in the Hackathon, but the time is not ripe yet, so I can only wish for the success of the event.

**Comment by ottlukas** [comment]
What if we do a PLC4X Developer Session online on 16 th in the evening instead?
